django_mongodb/features.py

```diff
@@ -202,6 +204,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             "annotations.tests.NonAggregateAnnotationTestCase.test_order_by_annotation",
             # annotate().filter().count() gives incorrect results.
             "db_functions.datetime.test_extract_trunc.DateFunctionTests.test_extract_year_exact_lookup",
+            "model_fields.test_jsonfield.TestQuerying.test_nested_key_transform_on_subquery",
+            "model_fields.test_jsonfield.TestQuerying.test_ordering_grouping_by_count",
```

Comment:
This test is "FieldDoesNotExist with ordering."

Reply:
What I meant is that there is already a section (a few lines above) with this name.